data/lib/core/constants.lua

```diff
@@ -30,3 +32,13 @@ SCREENSHOT_TYPE_SKILLUP = 12
 
 SCREENSHOT_TYPE_FIRST = SCREENSHOT_TYPE_ACHIEVEMENT
 SCREENSHOT_TYPE_LAST = SCREENSHOT_TYPE_SKILLUP
+
+BLESS_TYPE_ADVENTURER = 1 -- not supported
+BLESS_TYPE_TWIST_OF_FATE = 2 -- not supported
+BLESS_TYPE_WISDOM_OF_SOLITUDE = 4
+BLESS_TYPE_SPARK_OF_PHOENIX = 8
+BLESS_TYPE_FIRE_OF_THE_SUNS = 16
+BLESS_TYPE_SPIRITUAL_SHIELD = 32
+BLESS_TYPE_THE_EMBRACE_OF_THE_WORLD = 64
+BLESS_TYPE_HEART_OF_MOUNTAIN = 128 -- not supported
+BLESS_TYPE_BLOOD_OF_MOUNTAIN = 256 -- not supported
```

Comment:
Why are these not supported?

Reply:
not implemented in tfs, for they are the new blessings.